assets/airports/airportLoadList.js

```diff
@@ -43,7 +43,7 @@ window.AIRPORT_LOAD_LIST = (function() { // eslint-disable-line wrap-iife
      *    +----------+------------------------+
      *    |   Hard   |    More than 60 AcpH   |
      *    +----------+------------------------+
-     * - `AIRPORT_NAME` is the name of the airport (ex: San Francisco International Airport)
+     * - `AIRPORT_NAME` is the official English name of the airport (ex: San Francisco International Airport)
```

Comment:
Add 'according to Jeppesen charts'

Reply:
![Screenshot_255](https://user-images.githubusercontent.com/24833249/57581481-1c5a5f80-74b8-11e9-839a-b1f52bd7e0f9.png)
Well the thing with the Jeppesen charts is that it doesnt use the airports name its either the icao code (EDDM) or the city where the airport is located (MUNICH, GERMANY).

Reply:
Ah, I see. What source did you use for the names?

Reply:
Mainly the official website of the airport

Reply:
Hmm, I'm not sure whether that will work for every airport in the world

Reply:
It'd be better if we had a single source

Reply:
the problem is there isnt really a perfect source (or I haven't found one yet) that has all the official airport names. Maybe it would make more sense to leave it like that and decide for every airport separately in the review process. What do you think?

Reply:
Looks like you can still use this source. Just look at the airport diagram and use the name listed under the city. For EDDM, it just says "Munich", for LFPG "Charles-de-Gaulle", EDDF "Frankfurt/Main", KMIA "Miami Int'l". Should do the trick.